nservicebus/rabbitmq/routing-topology_support-delayed-delivery_rabbit_[4,].partial.md

```diff
@@ -0,0 +1,7 @@
+### Delayed delivery
+
+Starting with Version 4.3, it is possible to support [delayed delivery](delayed-delivery.md) without requiring the [timeout manager](/nservicebus/messaging/timeout-manager.md) by implementing `ISupportDelayedDelivery` in addition to `IRoutingTopology` in the routing topology class.
+
+When the routing topology implements `ISupportDelayedDelivery`, the transport will call the `BindToDelayInfrastructure` method to allow the routing topology to create the appropriate bindings to the delivery exchange that is part of the native delayed delivery infrastructure. 
+
+`IRoutingTopology` and `ISupportDelayedDelivery` will be combined into a single interface in Version 5.0.
```

Comment:
Looking at https://docs.particular.net/nservicebus/messaging/delayed-delivery#how-it-works a bit more, I don't think it makes sense to add something about RabbitMQ there. 

Instead, I'd say that entire section needs revising or removing, because despite the section title, it's not really "How it works" at all.

Regardless, I think making those changes are out of scope for this PR.